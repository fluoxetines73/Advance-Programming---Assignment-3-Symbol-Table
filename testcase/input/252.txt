RPRINT
PRINT
RPRINT
PRINT
RPRINT
BEGIN
RPRINT
RPRINT
INSERT z string
ASSIGN z 'UPPERlower'
PRINT
ASSIGN z 'abcABC'
RPRINT
RPRINT
INSERT a string
PRINT
BEGIN
END
END