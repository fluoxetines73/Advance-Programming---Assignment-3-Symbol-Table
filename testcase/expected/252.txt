






success
success
z//1
success
z//1
z//1
success
z//1 a//1